lxd/storage/drivers/utils.go

```diff
@@ -472,15 +472,15 @@ func growFileSystem(fsType string, devPath string, vol Volume) error {
 		case "ext4":
 			msg, err = shared.TryRunCommand("resize2fs", devPath)
 		case "xfs":
-			msg, err = shared.TryRunCommand("xfs_growfs", devPath)
+			msg, err = shared.TryRunCommand("xfs_growfs", mountPath)
 		case "btrfs":
 			msg, err = shared.TryRunCommand("btrfs", "filesystem", "resize", "max", mountPath)
 		default:
 			return fmt.Errorf(`Growing not supported for filesystem type "%s"`, fsType)
 		}
 
 		if err != nil {
-			return fmt.Errorf(`Could not extend underlying %s filesystem for "%s": %s`, fsType, devPath, msg)
+			return fmt.Errorf(`Could not extend underlying %s filesystem for "%s": %s (%s)`, fsType, devPath, msg, err)
```

Comment:
@monstermunchkin please can you use errors.Wrapf here rather than appending error in fmt.Errorf. Thanks

Reply:
@monstermunchkin as a side note, I've recently found that using `%q` as a placeholder rather than `"%s"` for strings will automatically quote the value with double quotes.